src/assets/scss/_global.scss

```diff
@@ -1,3 +1,10 @@
+@font-face {
+  font-family: 'gibsonlight';
```

Comment:
Confirmed with him. Sticking with Helvetica for consistency. Will remove the gibson references